class/api_subtotal.class.php

```diff
@@ -129,6 +129,14 @@ public function getTotalLine($elementtype, $idline = 1)
 
 	}
 
+
+    function calculate_price_for_total($parameters, &$object, &$action, $hookmanager)
+	{
+	    $basePRICE = getDolGlobalInt('Base_price');
+		return $object->total * $basePRICE
+	}
+
```

Comment:
6  :  Le  nom  de  la  fonction  ` calculate _price _for _total `  doit  être  en  camel Case  et  début er  par  un  ver be ,  par  exemple  ` calculate Price For Total `.